bn-expansions/src/calculateWeight.ts

```diff
@@ -0,0 +1,53 @@
+import { BaseCatch } from '@boatnet/bn-models';
+
+export function updateCatchWeight(weightMethod: number, catchVal: BaseCatch) {
+    if (weightMethod === 3) {
```

Comment:
could consider a switch statement instead of series of if/ else statements. If we have fallthrough cases (e.g. several weight methods using the same calculations) then a switch statement would be a little cleaner.

Reply:
ahhh good point updated to switch statement also created an enum to map numbers to weight methods